packages/react-native-performance/android/src/main/kotlin/com/shopify/reactnativeperformance/PerformanceMarker.kt

```diff
@@ -18,30 +18,21 @@ class PerformanceMarker(context: Context?) : View(context) {
 
     var destinationScreen: String? by PerformanceMarkerProp()
     var renderPassName: String? by PerformanceMarkerProp()
-    var interactive: Interactive? by PerformanceMarkerProp()
+    var interactive: Boolean? by PerformanceMarkerProp()
     var componentInstanceId: String? by PerformanceMarkerProp()
 
     init {
-        setWillNotDraw(true)
+        this.isEnabled = false;
     }
 
+    override fun onAttachedToWindow() {
+        super.onAttachedToWindow()
+        sendRenderCompletionEventIfNeeded()
+    }
+
+
     private var reportedOnce = false
 
-    /**
-      We're choosing to send the onRenderComplete event back to JS the moment the view instance is created
-      + the JS props are made available. We're not waiting until the view goes through the measure/layout/draw cycle.
-
-      This is because on Android, the view may not go through this cycle until it has an active window on which
-      it can paint itself. This means that if the corresponding JS component is not in focus, the native UI will
-      not actually paint itself immediately.
```

Comment:
maybe @naqvitalha  can explain better here, but my understanding is that ` the view may not go through this cycle until it has an active window on which it can paint itself` is not something we need to be concerned about here.